src/main/java/com/espot/employeemanagementapp/Entity/Employee.java

```diff
@@ -0,0 +1,29 @@
+package com.espot.employeemanagementapp.Entity;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name="Employee")
+public class Employee {
+    @Id
+    @JsonProperty("id")
+    Long id;
+    @JsonProperty("firstName")
+
+    String firstName;
+    @JsonProperty("lastName")
+
+    String lastName;
+    @JsonProperty("emailId")
+
+    String emailId;
+
+    public Employee(String firstName, String lastName, String emailId) {
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.emailId = emailId;
+    }
+}
```

Comment:
create Getter methods to access private fields for achieve encapsulation

Reply:
Done